### PR TITLE
Update Deno icon

### PR DESCRIPTION
**Pull request by LitoMore**
### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -4203,7 +4203,7 @@
         {
             "title": "Deno",
             "hex": "000000",
```
Comment (uncenter) [inline _data/simple-icons.json:4205]: Have we ever considered using `#70FFAF` as the hex here? They seem to use it as their accent/brand color in some places, though not necessarily in the logo itself.

CleanShot 2024-10-10 at 21 23 33

image

image

File: icons/deno.svg
Comment (uncenter) [inline icons/deno.svg:1]: Looks good!

File: _data/simple-icons.json (hunk 1 above)
Comment (LitoMore) [reply _data/simple-icons.json:4205]: We should not use that color on the icon. They didn't use this color on the logo.

Comment (uncenter) [reply _data/simple-icons.json:4205]: What do you mean the homepage theme color is subject to change... everything is subject to change? We are changing the "blank-and-white Deno" logo literally right now 😅

Comment (uncenter) [reply _data/simple-icons.json:4205]: And for the record [our guidelines say](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#color-guidelines) that the brand color isn't necessarily the color of the icon, just the accent color the brand uses in general.

Comment (LitoMore) [reply _data/simple-icons.json:4205]: They didn't use color on the logo. This is a color that has nothing to do with the icon, so we shouldn’t use it on the icon.

Comment (uncenter) [reply _data/simple-icons.json:4205]: Like I said, our guidelines don't say that the brand color must be from the icon. It looks pretty clear to me that Deno uses this green color as their "accent", so I don't see why not. We can always ask someone working at the company for confirmation too...

Comment (LitoMore) [reply _data/simple-icons.json:4205]: It's official is just black-and-white. They even added a white background to the icon to ensure that the icon is always black-and-white.

**Comment by LitoMore** [comment]
Hi @lucacasonato, could you help review this icon change? I believe the icon color should be always black-and-white. What do you think?

We had a similar discussion before at https://github.com/denoland/dotland/pull/1946.

**Review by PratyushJayachandran — APPROVED** [review]

**Comment by lucacasonato** [comment]
Either is acceptable as per our [logo guidelines](https://github.com/user-attachments/files/17339441/Logo.guidelines.pdf) (attached).

The preferred logo to use is the transparent background one. The one with an explicit background should only be used in exceptional cases. You can find all versions of the logo here: https://drive.google.com/drive/folders/14oL13lfBUZ6kAFkZy4GrE-IDVpr6HbGP

**Comment by LitoMore** [comment]
@lucacasonato Thank you. Then I would choose to use a colored one in case the user needs this color for other purposes.

**Pull request merged**